app/screens/select_server/select_server.js

```diff
@@ -73,12 +75,18 @@ export default class SelectServer extends PureComponent {
 
     constructor(props) {
         super(props);
+        const {allowOtherServers, serverUrl, deepLinkURL} = props;
+
+        let theUrl = serverUrl;
+        if (allowOtherServers && deepLinkURL) {
+            theUrl = 'https://' + urlParse(deepLinkURL).host;
```

Comment:
maybe remove the scheme and let the handler add it later, this does happen automatically.

for example if you type in the input `community.mattermost.com` it will try and connect to `https://community.mattermost.com` and then it will try with `http://community.mattermost.com`

Reply:
oh makes sense